app/src/components/ResourceCards.tsx

```diff
@@ -26,21 +27,18 @@ export default function ResourceCards({ resourceInfo, resourceType }: ResourceBu
   return (
     <div>
       <ScrollArea.Autosize mah={height * 0.8} type="scroll">
-        {resourceInfo.length > 0 ? (
-          <Stack
-            style={{
-              alignItems: 'center',
-              paddingBottom: '12px'
-            }}
-          >
-            {resourceInfo.map(res => {
-              return (
-                <div key={res.id}>
-                  <ResourceInfoCard resourceInfo={res} />
-                </div>
-              );
-            })}
-          </Stack>
+        {resourceInfo?.length > 0 ? (
+          <Center>
+            <Stack align="stretch" pb={2}>
```

Comment:
I think we can do what you did in `[resourceType].tsx` and change this to just be `Slack align="center"`, unless you want stretch but I am not entirely sure of its purpose. Also, I am not sure if `pb={2}` is necessary? I couldn't see any visual difference without it but maybe I am wrong!

Reply:
The visual difference with the bottom padding is very slight. It makes sure that the last card in the `Stack` does not get cut off (which essentially means that its shadow will still appear on the page).

Without the bottom padding (look at the card for `Measure/123`):
![Screenshot 2023-06-16 at 1 38 42 PM](https://github.com/projecttacoma/measure-repository/assets/87094479/044b4206-a02b-4cf4-b34d-887db6bc682f)

With the bottom padding:
![Screenshot 2023-06-16 at 1 39 18 PM](https://github.com/projecttacoma/measure-repository/assets/87094479/5a56f8f0-40ab-47d6-aed9-cd5d27f76f18)

With `align="center"`, this is what happens sometimes when working with a condensed screen size:
![Screenshot 2023-06-16 at 2 39 43 PM](https://github.com/projecttacoma/measure-repository/assets/87094479/f85caf39-9349-4f39-a6ec-00e73750056e)

For the most part we won't see this I guess, but I found that using `align="stretch"` stretches the resource cards to take up the entire width so that they are uniform width no matter the screen size:
![Screenshot 2023-06-16 at 2 41 23 PM](https://github.com/projecttacoma/measure-repository/assets/87094479/c8ba3944-0565-402c-9b8d-517fe8fded33)